backend/ticketvise/models/ticket.py

```diff
@@ -106,34 +112,51 @@ def save(self, force_insert=False, force_update=False, using=None, update_fields
         assignee. Also send an email to the student that submitted the ticket if the status has changed to assigned
         or closed. Only send the change to assigned email if show assignee is enabled for the inbox.
         """
+        old_ticket = None
         old_status = None
 
         if self.id:
-            old_status = Ticket.objects.get(pk=self.id).get_status()
+            old_ticket = Ticket.objects.get(pk=self.id)
+            old_status = old_ticket.status
         else:
             # + 1 so the ticket_inbox_id starts at 1 instead of 0.
             self.ticket_inbox_id = Ticket.objects.filter(inbox=self.inbox).count() + 1
 
         super().save(force_insert, force_update, using, update_fields)
 
+        if old_ticket:
+            if old_ticket.status != self.status:
+                TicketStatusEvent.objects.create(ticket=self, initiator=get_current_authenticated_user(),
+                                                 old_status=old_status, new_status=self.status)
+
+            if old_ticket.title != self.title:
+                TicketTitleEvent.objects.create(ticket=self, old_title=old_ticket.title, new_title=self.title,
+                                                initiator=get_current_authenticated_user())
```

Comment:
wanneer zou een title van een ticket veranderen, volgens mij is dat nu niet aan de orde dat je het kan editen